### PR TITLE
1916540: Negative proxy tests occasionally encounter the wrong except…

### PR DESCRIPTION
**Pull request by tlhmerry9**
…ion handling

### CODE REVIEW

File: src/subscription_manager/cache.py [hunk 1]
```diff
@@ -56,7 +56,9 @@ class CacheManager(object):
     changed on subsequent runs.
     """
 
+
```
Comment (jirihnidek) [inline src/subscription_manager/cache.py:59]: Too many blank lines.

**Review by jirihnidek — CHANGES_REQUESTED** [review]
Hi,
Thanks for the PR. I can confirm that this change fixes the problem, but it breaks some unit tests. Please fix them. There are some stylish errors.

**Comment by tlhmerry9** [comment]
Fixed the tests related to this change.  

This is blocked by PR [#2479](https://github.com/candlepin/subscription-manager/pull/2479).   Once PR [#2479](https://github.com/candlepin/subscription-manager/pull/2479) is merged with master this branch needs to be rebased to master to resolve the test errors.

File: src/subscription_manager/cache.py (hunk 1 above)
Comment (tlhmerry9) [reply src/subscription_manager/cache.py]: done

File: src/subscription_manager/cache.py [hunk 2]
```diff
@@ -57,6 +57,7 @@ class CacheManager(object):
     """
 
     # Fields the subclass must override:
+
```
Comment (ptoscano) [inline src/subscription_manager/cache.py:60]: extra empty line addition

Comment (tlhmerry9) [reply src/subscription_manager/cache.py]: removed

**Comment by tlhmerry9** [comment]
@candlepin-bot ok to test

**Comment by tlhmerry9** [comment]
@candlepin-bot please retest this

**Comment by tlhmerry9** [comment]
all requested changes have been made

**Review by jirihnidek — APPROVED** [review]

**Pull request merged**